modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -118,10 +115,7 @@ definition {
 	@priority = 3
 	test CanDeleteAttachment {
 		task ("Given an object with an attachment field is created") {
-			ObjectAdmin.addObjectViaAPI(
-				labelName = "Custom Object 161928",
-				objectName = "CustomObject161928",
-				pluralLabelName = "Custom Objects 161928");
```

Comment:
Don't replace the `addObjectViaAPI` to `Notifications.notificationTemplateAdmin();`